easypost/address.py

```diff
@@ -20,21 +21,19 @@ class Address(AllResource, CreateResource):
     def create(
         cls,
         api_key: Optional[str] = None,
-        verify: Optional[Dict[str, Any]] = None,
-        verify_strict: Dict[str, Any] = None,
+        verify: Optional[Union[Dict[str, Any], str, bool]] = None,
```

Comment:
Thought: I hate this. Shouldn't we be driving people to only providing a boolean (especially in Python, where this type hint is just a suggestion and not actually enforced)?

Reply:
We can't break backwards compatibility or expectations. But yes, over a long period of time we can. First step is updating the docs to say "use a bool" which is done. Then in an upcoming major release we could start restricting this. But almost everyone is using arrays.

Reply:
I understand. I was just saying, we could start "restricting" it now, since it's not actually restricted (the type hint can say "you have to provide a boolean", but users could continue giving it an array and it won't actually break, yay non-type-strict languages)

Reply:
Ah. Well because we provide these type hints to users who import the library, it would complain if they also use mypy which would then force them to ignore a warning (not ideal) or make a code change on a non-major version bump.

And also, even though we may only "want" users to do a bool, the others are still valid options.